pexels_scraper.py

```diff
@@ -1,7 +1,20 @@
 from base_scraper import BaseScraper
+from selenium.webdriver.support.ui import WebDriverWait
 
 
 class PexelsScraper(BaseScraper):
+    website = "https://www.pexels.com/search/headshot/"
+    image_repository_bucket = "scraped-images-b1"
+
     def __init__(self):
-        super().__init__()
+        super().__init__(self.website, self.image_repository_bucket)
         print("PexelsScraper init")
+
+    def scrape_images(self):
+        print("PexelScarper scrape_images")
+        try:
+            browser = self.init_web_driver()
+            wait = WebDriverWait(browser, 10)
+            self.save_pexel_images(browser, wait, self.website)
+        except Exception as e:
+            print("Exception", e)
```

Comment:
what about the `scrape_next_images` function ? How is pagination/infinite scrolling handled ?

Reply:
@abhi526691 can you add the scrape_next_images which calls scroll from the `BaseScraper` ?